invoices/invoices_test.go

```diff
@@ -198,12 +200,30 @@ func TestInvoices(t *testing.T) {
 		return db
 	}
 
+	makeSQLDB := func(t *testing.T) invpkg.InvoiceDB {
+		db := sqldb.NewTestSqliteDB(t)
```

Comment:
`postgres` as well? May find some edge cases where all the queries aren't 100% portable.

Reply:
Added in a separate commit.